backend.py

```diff
@@ -435,6 +438,9 @@ def online(self):
 
     @property
     def intrinsics(self):
+        if not self.online:
+            return None
```

Comment:
Returning `None` might result in raised exceptions as well. Best would be to return a dummy camera model set to `self.frame_size`